RocketChat/Rocket.Chat
# fix: "Media type not accepted" error when uploading `lst` files

## Proposed changes (including videos or screenshots)

 - Add missing mime type for `.lst` files.

## Issue(s)


## Steps to test or reproduce

Download the sample lst file used in our end-to-end tests and try uploading it. No error should be trown.

## Further comments


[SUP-518](https://rocketchat.atlassian.net/browse/SUP-518?atlOrigin=eyJpIjoiY2Q1ZWE5MTBiN2FkNDdmOWFjMDYyNDcwYjM2ZDVjY2YiLCJwIjoiaiJ9)

[SUP-518]: https://rocketchat.atlassian.net/browse/SUP-518?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
